sp800-63d/vot_mapping.md

```diff
@@ -118,25 +118,87 @@ FAL maps into the VoT component A using a numeric identifier. Only one numeric i
-
 |Presentation|Component Value|
 |:----:|:--:|
 |Front channel|Af|
 |Back channel|Ab|
 |Additional features implemented beyond requirements for asserted FAL|Ax|
 
-Any discrepancy in the assessed and implemented assurance level SHOULD be documented in a digital identity acceptance statement, per SP 800-63-3 Section 5.5.
+## How to Use the Values in Vectors
 
-### Authenticator Management to VoT
+The implemented IAL, AAL, and/or FAL component value(s) SHALL be indicated in the vector. Only one numeric identifier SHALL be used for each xAL. Any discrepancies in the assessed and implemented assurance level SHOULD be documented in a digital identity acceptance statement, per SP 800-63-3 Section 5.5.
 
-Authenticator lifecyle management, per 800-63B Section 6, maps into the VoT component M using an alphabetic identifier. Authenticator management method MAY be indicated using an alphabetic identifier. Multiple alphabetic identifiers MAY be used simultaneously.
+Identity proofing characteristics, authenticator type(s) used, authentication characteristics, authenticator lifecycle management characteristics, and assertion presentation values MAY be indicated in the vector. Per the requirements in the following sections, some of these values SHALL be indicated in the vector depending on the approaches implemented. Multiple alphabetic identifiers MAY be used simultaneously. 
 
-|Credential Management Method|Component Value|
-|:----:|:--:|
-|Credential issued/bound during proofing session|Mp|
-|Credential issued/bound after proofing session (remote)|Mr|
-|Credential issued/bound after proofing session (in-person)|Mi|
-|Second factor bound to a single-factor account|Ms|
-|Authentication factors reestablished (account recovery) using full proofing process|Ma|
-|Authentication factors reestablished (account recovery) using abbreviated proofing process|Mb|
\ No newline at end of file
+Identity proofing characteristics, authenticator type(s) used, authentication characteristics, and assertion presentation values SHOULD be indicated only when they demonstrate the implementation of approaches beyond those required by the xAL indicated in the vector. The appropriate value that signifies this (i.e., Px, Cx, and Ax) SHOULD also be indicated. Values SHOULD NOT be indicated that represent approaches already required by the xAL indicated in the vector. These approaches are assumed to be implemented per the xAL's requirements. 
+
+**Example:** C2.Cx.Cv could be asserted if implementing AAL2 and an authenticator is used that provides verifier impersonation resistance, since verifier impersonation resistance is not required under AAL2.
+
+Any additional component categories SHALL NOT be used.
+
+### Identity Proofing Characteristics
+
+If implementing IAL2 and KBV is used for identity verification, Pk SHALL be indicated in the vector (e.g., P2.Pk).
```

Comment:
Is there reason for these normative requirements to be in another section? Seems like this should be bundled with the definitions above, but maybe I'm missing something in the structure.